src/app/shared/components/addregressionregion/addregressionregion.component.ts

```diff
@@ -310,6 +338,78 @@ export class AddRegressionRegionModal implements OnInit, OnDestroy {
+              this.modalRef.close();
+            } else {this._settingsService.outputWimMessages(res); }
+            if (this.addCitation && this.selectedRegRegion.citationID) { // Editing citation
+                this._settingsService.putEntity(this.selectedRegRegion.citationID, this.newCitForm.value, this.configSettings.citationURL)
+                .subscribe((response: any) => {
+                  if (!response.headers) { // Citation successfully updated
+                    this._toasterService.pop('info', 'Info', 'Citation was updated');
+                  } else { 
+                    this._settingsService.outputWimMessages(response); 
+                    this.cancelCreateRegression(); 
+                  }
+                  this.requeryFilters();
+                  this.cancelCreateRegression();
+                }, error => {
+                  this._loaderService.hideFullPageLoad();
+                  if (this._settingsService.outputWimMessages(error)) { return; }
+                  this._toasterService.pop('error', 'Error creating Citation', error.message || error._body.message || error.statusText);
+                });
+            } else if (this.addCitation) { // New citation
+             
+              if (this.newCitation == true) {
+                this.createNewCitation(this.selectedRegRegion);
+              } else {
+                this._loaderService.hideFullPageLoad();
+                this.requeryFilters();
+                this.cancelCreateRegression();
+              }
+            } else {
+                this.requeryFilters();
+                this.cancelCreateRegression();
+            }
+        }, error => {
+            this._loaderService.hideFullPageLoad();
+            if (this._settingsService.outputWimMessages(error)) {return; }
+            this._toasterService.pop('error', 'Error editing Regression Region', error.message || error._body.message || error.statusText); }
+        );  
+  }
+
+  public removeCitation(){
+    this.saveFilters();
+        const idx = this.regressionRegions.findIndex(r => r.id === this.rr);
+        const regReg = this.regressionRegions[idx];
+        regReg.citationID = null;
+        this._settingsService.putEntity(this.rr, regReg, this.configSettings.regRegionURL)
+            .subscribe((response) => {
+                this.requeryFilters();
+                this._nssService.outputWimMessages(response);
+            }, error => {
+                if (this._settingsService.outputWimMessages(error)) {return; }
+                this._toasterService.pop('error', 'Error removing Citation', error._body.message || error.statusText);
+            }
+        );
+        this.selectedRegRegion.citationID=null;
+        this.newRegRegForm.controls['citationID'].setValue(null);
+        this.newCitForm.controls['title'].setValue(null);
+        this.newCitForm.controls['author'].setValue(null);
+        this.newCitForm.controls['citationURL'].setValue(null);
```

Comment:
Resetting the `newCitForm` (line 410) should reset all the controls in it, so this isn't needed.